common/sys_serenity.c

```diff
@@ -373,7 +373,7 @@ main(int argc, const char *argv[])
     Host_Init(&parms);
 #endif /* SERVERONLY */
 
-    if (pledge("stdio thread shared_buffer rpath wpath cpath", NULL) < 0) {
+    if (pledge("stdio thread rpath wpath cpath", NULL) < 0) {
```

Comment:
We still need the `recvfd` and `sendfd` pledges I think. In case the GUI theme or system font changes for example.

Reply:
The game was running the first few levels without this, but of course I only tested a fraction of the game.
I added the pledges you suggested now.
Thanks for the swift reply :)